Composer/packages/client/src/shell/actionApi.ts

```diff
@@ -126,6 +132,7 @@ export const useActionApi = (projectId: string) => {
   }
 
   async function deleteAction(dialogId: string, action: MicrosoftIDialog) {
+    TelemetryClient.log('ActionDeleted', { type: action.$kind });
```

Comment:
Consider just calling `deleteActions` like `constructAction` does and remove this.